test/content_test.exs

```diff
@@ -1,26 +1,14 @@
 defmodule ContentTest do
   use ExUnit.Case
   use Plug.Test
-  # @opts AuthPlug.init(%{auth_url: "https://dwylauth.herokuapp.com"})
```

Comment:
Thanks for removing this. ✂️ 
I obviously borrowed code directly from https://github.com/dwyl/auth_plug 😉